test/load-balancer-mocha.js

```diff
@@ -0,0 +1,90 @@
+// load-balancer-mocha.js
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.should();
+chai.use(chaiAsPromised);
+
+var http = require('http');
+var _ = require('underscore');
+var os = require('os');
+var fs = require('fs');
+var path = require('path');
+var noflo = require('noflo');
+var test = require('./common-test');
+
+describe('load-balancer', function() {
+    var port = 1337;
+    it("should POST input to single target", function() {
+        return new Promise(function(done, fail){
```

Comment:
I'm guessing this test is going to need more than the default 2 sec to run when we execute it as part of the mocha suite.  Have you run the full test suite to see if this test completes in time?